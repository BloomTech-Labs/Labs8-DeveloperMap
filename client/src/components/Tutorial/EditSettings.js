import React from 'react';
import styled from 'styled-components';


const EditSettings = () => {
    return (
        <SettingsGuide>
           <h1>Edit Settings</h1>
           <p>To edit your profile settings and change your information, click on the cog in the top 
               right of your screen.
           </p>
           <h1>Picture goes here</h1>
           <p>Then hit edit in the top right corner of the modal. Make sure you hit the save button before 
               closing, or your new settings will not be saved.
           </p>
           <h1>Picture goes here</h1>
        </SettingsGuide>
    );
}

const SettingsGuide = styled.div`
  padding: 5%;
`;

export default EditSettings;