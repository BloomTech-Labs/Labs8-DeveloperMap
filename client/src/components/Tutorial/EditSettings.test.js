import React from 'react';
import renderer from 'react-test-renderer';
import EditSettings from './EditSettings';
import { shallow } from 'enzyme';

describe(`edit settings Component`, () => {
    it('renders correctly', () => {
        const tree = shallow(
          <EditSettings />
        );
        expect(tree).toMatchSnapshot();
      });

});