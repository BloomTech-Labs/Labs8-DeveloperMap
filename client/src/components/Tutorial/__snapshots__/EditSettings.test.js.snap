// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`edit settings Component renders correctly 1`] = `
<styled.div>
  <h1>
    Edit Settings
  </h1>
  <p>
    To edit your profile settings and change your information, click on the cog in the top right of your screen.
  </p>
  <img
    alt=""
    src="favicon.PNG"
  />
  <p>
    Then hit edit in the top right corner of the modal. Make sure you hit the save button before closing, or your new settings will not be saved.
  </p>
  <img
    alt=""
    src="edit.PNG"
  />
  <styled.button
    onClick={[Function]}
  >
    Done
  </styled.button>
</styled.div>
`;
